### ISSUE TITLE
time to train a model

### ISSUE THREAD

**Issue by lyltencent** [endernewton/tf-faster-rcnn#84]
Hi , 

I am just wondering how long does it take to train my own model using the following command: (using GPU, PASCAL) 

./experiments/scripts/train_faster_rcnn.sh 0 pascal_voc vgg16
./experiments/scripts/train_faster_rcnn.sh 1 coco res101

I have tried to run the following code now. 
./experiments/scripts/train_faster_rcnn.sh 0 pascal_voc  res101

However, it has been almost 4 days and still has not finished yet. 
Thanks,

**Comment by endernewton** [comment]
what is your gpu? how fast is each iter on your side?

**Comment by lyltencent** [comment]
Hi Xinlei, @endernewton , the GPU I have is the TITAN X (PASCAL) with 12 GB memory. 
I also note that in your tutorial, you mention about the following, 

Note: double check you have deleted softlink to the pre-trained models before training!

It seems that I din't do this step, will that be the reason ?

Thanks,

**Comment by supertuzi** [comment]
I have similar issue here. GPU(1080) Utilization almost zero. 
./experiments/scripts/train_faster_rcnn.sh 0 coco res101
iter: 20 / 490000, total loss: 1.720961
 >>> rpn_loss_cls: 0.689545
 >>> rpn_loss_box: 0.557195
 >>> loss_cls: 0.474220
 >>> loss_box: 0.000000
 >>> lr: 0.001000
speed: 19.982s / iter

**Comment by endernewton** [comment]
you need to install your tensorflow with gpu support

---------------------
Ender X. Chen

Language Technologies Institute,
School of Computer Science,
Carnegie Mellon University, Pittsburgh, PA

**Comment by lyltencent** [comment]
Thanks Xinlei, @endernewton . I think that is the silly mistake I made...I am going to close this issue for now...
